website/developers/links.md

```diff
@@ -1,20 +1,18 @@
 ---
 layout: website-normal
-title: Development Bookmarks
-menu_proxy_for: index.md
+title: Handy Places
 ---
 
 {% include fields.md %}
 
-Handy places:
-
 * **Code** is in Github at [https://github.com/apache/brooklyn/](https://github.com/apache/brooklyn/)
 
 * **Issues** are in Jira at [https://issues.apache.org/jira/browse/BROOKLYN/](https://issues.apache.org/jira/browse/BROOKLYN/)
 
-* **Maven repositories** are:
+* **Maven repositories** are at:
+
   * [Apache releases]({{ apache_releases_repo_groupid_url }})
   * [Apache snapshots]({{ apache_snapshots_repo_groupid_url }})
- 
-* **CI server** is a jenkins server at: [https://builds.apache.org/job/brooklyn-master-build/](https://builds.apache.org/job/brooklyn-master-build/)
+
+* **CI** is done by Jenkins at [https://builds.apache.org/job/brooklyn-master-build/](https://builds.apache.org/job/brooklyn-master-build/)
```

Comment:
I think this file can be dropped completely. GitHub and JIRA links are duplicated information as they are already in the menu; Jenkins isn't but it should be. There shouldn't be any need to refer to the Apache releases Maven repo as it pushes to Maven Central. That just leaves the Apache snapshot repo, and I expect we can find a better place to document that.